### DISCUSSION TITLE
mitmproxy cannot parse HTTP response

### DISCUSSION THREAD

**Discussion by AuQuangDuc** [mitmproxy/mitmproxy#7062] [General]
In my web application, I have a response:

![image](https://github.com/user-attachments/assets/22580e1b-944f-4b21-9782-fdfb8b71dc18)

However, mitmproxy is showing an error:

![image](https://github.com/user-attachments/assets/a7e964a5-2170-4249-ad47-f58cf283320d)

Please help me

**Comment by Prinzhorn** [comment]
The error message is pretty clear, this is not valid HTTP. Fix your web application. 

curl would give you:

```
curl: (8) Header without colon
```
